src/Functions/Public/relocation/New-vRAOnboardingPlan.ps1

```diff
@@ -112,7 +112,7 @@
                 $URI = "/relocation/onboarding/plan"
                 $OnboardingPlan = Invoke-vRARestMethod -Method POST -URI $URI -Body $Body -Verbose:$VerbosePreference
 
-                CalculateOutput
+                CalculateOutput($OnboardingPlan)
```

Comment:
Should the parameter be positional here and not in ()?

Reply:
resolved in last commit. :)